utils/range.ts

```diff
@@ -0,0 +1,7 @@
+export const range = ({
```

Comment:
Is this just a range helper function?

Reply:
Ya, for whatever reason JavaScript doesn't have a built-in one and it's used in the pagination hook. Followed [this article](https://www.freecodecamp.org/news/javascript-range-create-an-array-of-numbers-with-the-from-method/).